dev-packages/e2e-tests/test-applications/node-profiling/build.mjs

```diff
@@ -0,0 +1,17 @@
+// Because bundlers can now predetermine a static set of binaries we need to ensure those binaries
+// actually exists, else we risk a compile time error when bundling the package. This could happen
+// if we added a new binary in cpu_profiler.ts, but forgot to prebuild binaries for it. Because CI
+// only runs integration and unit tests, this change would be missed and could end up in a release.
+// Therefor, once all binaries are precompiled in CI and tests pass, run esbuild with bundle:true
+// which will copy all binaries to the outfile folder and throw if any of them are missing.
+import esbuild from 'esbuild';
+
+esbuild.build({
+  platform: 'node',
+  entryPoints: ['./index.js'],
+  outdir: './dist',
+  target: 'esnext',
+  format: 'cjs',
+  bundle: true,
+  // loader: { '.node': 'copy' },
```

Comment:
This needs to be enabled before the PR is merged. With this, change, esbuild will correctly try and copy all the binaries to the outdir and correctly fail if some cannot be resolved.